RELEASE-NOTES.md

```diff
@@ -19,6 +19,7 @@
 * `--temporary-connection` flag, that allows you to connect, without anything declared in config file
 * `snow streamlit init` command that creates a new streamlit project.
 * `snow streamlit deploy` support pages and environment.yml files.
+* Added project definition for Streamlit
```

Comment:
I'm not sure if it is only a new addition. What about removing the flags? It's not backward-compatible.

Reply:
Added

Reply:
IMO in `Backward incompatibility` we should not only mention option removals but also the requirement of having `snowflake.yml` in a directory with a streamlit project.

Reply:
I think it is ok, `snowflake.yml` is new

Reply:
Discussed and agreed that the requirement of `snowflake.yml` should be mentioned in "backward incompatibility"